src/swoop/api/io.py

```diff
@@ -0,0 +1,107 @@
+import io
+import logging
+
+from minio import Minio
+from minio.error import S3Error
+
+logger = logging.getLogger("uvicorn")
+logger.setLevel(logging.INFO)
+
+
+class IOClient:
+    client = None
+    bucket_name = None
```

Comment:
I don't think there's a good reason to make these class attributes, so I'd consider deleting them from the class.

```suggestion
```